src/Widgets/DataLoadMoreButtonWidget/DataLoadMoreButtonWidgetComponent.tsx

```diff
@@ -44,10 +39,7 @@ provideComponent(DataLoadMoreButtonWidget, ({ widget }) => {
 
   return (
     <WidgetTag className={alignmentClassName}>
-      <button
-        className={classNames.join(' ')}
-        onClick={() => setLimit(limit + 10)}
-      >
+      <button className={classNames.join(' ')} onClick={() => loadMore()}>
```

Comment:
* [ ] FYI could be simplified to `<button className={classNames.join(' ')} onClick={loadMore}>`